cookbooks/cdo-ruby/.kitchen.yml

```diff
@@ -6,9 +6,10 @@ provisioner:
   name: chef_zero
   data_path: test/shared
   require_chef_omnibus: 17.6.18
-
+verifier:
+  name: inspec
```

Comment:
Re-enable verification tests for `cdo-ruby`: previously our tests were disabled, so chef never verified the versions it tried to install.